packages/friends/yml/en/sarah-grubb.yml

```diff
@@ -5,7 +5,7 @@ gender: female
 born: 1756
 died: 1790
 quotes:
-  - source: Samuel Neale
+  - source: James Gough
```

Comment:
I’m not clear on this change -- the quote I think is about Mary Peisley, -- it was sort of one of my fake quotes to demonstrate the schema. I think to fully correct it, the whole quote would need to be removed.